src/main/java/com/google/sps/taskqueue/push/BlobstoreTaskWorker.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.taskqueue.push;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/blobstoreWorker")
+public class BlobstoreTaskWorker extends HttpServlet {
+
+  protected void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    String key = request.getParameter("key");
+
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    BlobKey blobKey = new BlobKey(key);
+    try {
+      /* Blobstore doesn't have a transaction as a part of its API
```

Comment:
Use proper grammar in comments (period).

Reply:
done